### DISCUSSION TITLE
Webhooks [Jul 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2993] [Discord Threads]
> 🙋‍♂️ **Elf**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@160755405033701377> You can find details here   
#### https://docs.nocodb.com/developer-resources/webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1001282414426984479)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
